Eevee.Sleep.Bot/Controllers/InGameAnnouncementController.cs

```diff
@@ -1,30 +1,35 @@
 using Eevee.Sleep.Bot.Controllers.Mongo.InGameAnnouncement;
+using Eevee.Sleep.Bot.Controllers.Mongo.InGameAnnouncement.Officialsite;
 using Eevee.Sleep.Bot.Enums;
+using Eevee.Sleep.Bot.Models.InGameAnnouncement.Officialsite;
 using Microsoft.AspNetCore.Mvc;
 
 namespace Eevee.Sleep.Bot.Controllers;
 
 [ApiController]
 [Route("/game/announcement")]
-public class InGameAnnouncementController(
-) : ControllerBase {
+public class InGameAnnouncementController() : ControllerBase {
```

Comment:
Name this to just `AnnouncementController` instead to keep the "neutrality" of this controller
Implementation detail decides the announcement source to grab